ui/server/src/test/scala/pl/touk/nussknacker/ui/definition/UIProcessObjectsSpec.scala

```diff
@@ -56,17 +72,39 @@ class UIProcessObjectsSpec extends FunSuite with Matchers {
         FixedExpressionValue("'a'", "a"),
         FixedExpressionValue("'b'", "b"),
         FixedExpressionValue("'c'", "c")
-      )))
+      ))),
+      "param3" -> None,
+      "param4" -> None
     )
 
 
     processObjects.nodesToAdd.find(_.name == "enrichers")
       .flatMap(_.possibleNodes.find(_.label == "enricher"))
       .map(_.node.asInstanceOf[Enricher].service.parameters) shouldBe Some(List(Parameter("param", Expression("spel", "'default value'")),
-      Parameter("param2", Expression("spel", "'a'"))))
+      Parameter("param2", Expression("spel", "'a'")),
+      Parameter("param3", Expression("spel", "''")),
+      Parameter("param4", Expression("spel", "''"))
+    ))
 
   }
 
+  test("should read editor from config") {
```

Comment:
1. Can you also add some test in `*ResourcesSpec` with returned json checking? It will show how api looks like and will test feature end-to-end (also json encoding).
2. Can you add some editor to our sample model that we can test it manually? Take a look at `DevProcessConfigCreator.SimpleTypesService`

Reply:
Done

Reply:
nice :+1: